### ISSUE TITLE
kernel died after some code 

### ISSUE THREAD

**Issue by SlowMonk** [tensorflow/models#6770]
from __future__ import absolute_import, division, print_function, unicode_literals

OS Platform and Distribution : ubuntu 18.10
TensorFlow installed from : !pip install tensorflow-gpu==2.0.0-alpha0
TensorFlow version : 2.0
Bazel version :?
CUDA/cuDNN version : 10.0
GPU model and memory : titan xp 12g
Exact command to reproduce

kernel died after some step
cuda 10.0 
tensorflow 2.0 
ubuntu 18.10 
titan xp 
running on jupyter notebook
==============================================
import glob
import imageio
import matplotlib.pyplot as plt
import numpy as np
import os
import PIL
from tensorflow.keras import layers
import time

from IPython import display
import tensorflow as tf

BUFFER_SIZE = 60000
BATCH_SIZE = 256 
(X_train,y_train),(X_test,y_test)= tf.keras.datasets.mnist.load_data()
X_train = X_train.reshape(X_train.shape[0],28,28,1).astype('float32')
X_train, y_train = (X_train/-127.5)/127.5 , (y_train-127.5)/127.5
print(X_train.shape)

#Batch and shuffle the data 
train_dataset = tf.data.Dataset.from_tensor_slices(X_train).shuffle(BUFFER_SIZE).batch(BATCH_SIZE)

def make_generator_model():
    model = tf.keras.Sequential()
    model.add(layers.Dense(7*7*256, use_bias=False, input_shape=(100,)))
    model.add(layers.BatchNormalization())
    model.add(layers.LeakyReLU())

    model.add(layers.Reshape((7, 7, 256)))
    assert model.output_shape == (None, 7, 7, 256) # Note: None is the batch size

    model.add(layers.Conv2DTranspose(128, (5, 5), strides=(1, 1), padding='same', use_bias=False))
    assert model.output_shape == (None, 7, 7, 128)
    model.add(layers.BatchNormalization())
    model.add(layers.LeakyReLU())

    model.add(layers.Conv2DTranspose(64, (5, 5), strides=(2, 2), padding='same', use_bias=False))
    assert model.output_shape == (None, 14, 14, 64)
    model.add(layers.BatchNormalization())
    model.add(layers.LeakyReLU())

    model.add(layers.Conv2DTranspose(1, (5, 5), strides=(2, 2), padding='same', use_bias=False, activation='tanh'))
    assert model.output_shape == (None, 28, 28, 1)

    return model

generator = make_generator_model()

noise = tf.random.normal([1, 100])
generated_image = generator(noise, training=False)

plt.imshow(generated_image[0, :, :, 0], cmap='gray')
================================================================
[I 09:22:47.585 NotebookApp] Saving file at /Gits/oneShot/DCGAN_practice.ipynb
2019-05-13 09:23:28.472619: I tensorflow/stream_executor/platform/default/dso_loader.cc:43] Successfully opened dynamic library libcuda.so.1
2019-05-13 09:23:28.630156: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1595] Found device 0 with properties: 
name: TITAN Xp major: 6 minor: 1 memoryClockRate(GHz): 1.582
pciBusID: 0000:42:00.0
totalMemory: 11.90GiB freeMemory: 10.98GiB
2019-05-13 09:23:28.630193: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1718] Adding visible gpu devices: 0
2019-05-13 09:23:28.630606: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2019-05-13 09:23:28.651496: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x5643203ef460 executing computations on platform CUDA. Devices:
2019-05-13 09:23:28.651543: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): TITAN Xp, Compute Capability 6.1
2019-05-13 09:23:28.672430: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 3493175000 Hz
2019-05-13 09:23:28.673866: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x5643207fab30 executing computations on platform Host. Devices:
2019-05-13 09:23:28.673895: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): <undefined>, <undefined>
2019-05-13 09:23:28.674002: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1718] Adding visible gpu devices: 0
2019-05-13 09:23:28.674275: I tensorflow/stream_executor/platform/default/dso_loader.cc:43] Successfully opened dynamic library libcudart.so.10.0
2019-05-13 09:23:28.675592: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1126] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-05-13 09:23:28.675613: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1132]      0 
2019-05-13 09:23:28.675622: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1145] 0:   N 
2019-05-13 09:23:28.675959: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1266] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 10677 MB memory) -> physical GPU (device: 0, name: TITAN Xp, pci bus id: 0000:42:00.0, compute capability: 6.1)
2019-05-13 09:23:29.749328: I tensorflow/stream_executor/platform/default/dso_loader.cc:43] Successfully opened dynamic library libcublas.so.10.0
2019-05-13 09:23:29.853357: I tensorflow/stream_executor/platform/default/dso_loader.cc:43] Successfully opened dynamic library libcudnn.so.7
2019-05-13 09:23:30.245744: E tensorflow/stream_executor/cuda/cuda_dnn.cc:319] Loaded runtime CuDNN library: 7.3.1 but source was compiled with: 7.4.2.  CuDNN library major and minor version needs to match or have higher minor version in case of CuDNN 7.0 or later version. If using a binary install, upgrade your CuDNN library.  If building from sources, make sure the library loaded at runtime is compatible with the version specified during compile configuration.
2019-05-13 09:23:30.247192: E tensorflow/stream_executor/cuda/cuda_dnn.cc:319] Loaded runtime CuDNN library: 7.3.1 but source was compiled with: 7.4.2.  CuDNN library major and minor version needs to match or have higher minor version in case of CuDNN 7.0 or later version. If using a binary install, upgrade your CuDNN library.  If building from sources, make sure the library loaded at runtime is compatible with the version specified during compile configuration.
2019-05-13 09:23:30.247689: F tensorflow/core/kernels/conv_grad_input_ops.cc:946] Check failed: stream->parent()->GetConvolveBackwardDataAlgorithms( conv_parameters.ShouldIncludeWinogradNonfusedAlgo<T>(stream->parent()), &algorithms) 
[I 09:23:32.606 NotebookApp] KernelRestarter: restarting kernel (1/5), keep random ports
WARNING:root:kernel 031ce406-02ec-48b7-aadc-28512c22f001 restarted

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by SlowMonk** [comment]
I updated the above information and 
I never installed another version of cudnn I installed before but error message keep saying that 
"2019-05-14 09:43:41.710155: E tensorflow/stream_executor/cuda/cuda_dnn.cc:328] Loaded runtime CuDNN library: 7.3.1 but source was compiled with: 7.4.2.  CuDNN library major and minor version needs to match or have higher minor version in case of CuDNN 7.0 or later version. "
so I should check the which version of cudnn library current tensorflow are using. 
Do you guys have any idea about this?

**Comment by zhilaAI** [comment]
I had the same issue, when number of images is big it tends to die. so I decrease the number of testing images and it works.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be an old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
